cmd/agent/app/reporter/flags.go

```diff
@@ -32,20 +32,28 @@ const (
 	GRPC Type = "grpc"
 
 	agentTags = "agent.tags"
+
+	//tags to be included for sending to collector
+	filterProcessTags = "filterProcessTags"
```

Comment:
"filter" is ambiguous. This could be called `agent.drop-process-tags` (later paired with `agent.drop-span-tags`).